core/__tests__/actions/schedules.ts

```diff
@@ -245,6 +245,53 @@ describe("actions/schedules", () => {
         expect(schedule.recurringFrequency).toBe(0);
       });
 
+      test("an administrator can edit refresh enabled", async () => {
```

Comment:
Do we need similar tests for `recurring`?

Reply:
These were setup in #2828